pkgs/native_toolchain_c/lib/src/cbuilder/cbuilder.dart

```diff
@@ -57,18 +57,27 @@ class CBuilder implements Builder {
   @visibleForTesting
   final Uri? installName;
 
+  /// Whether the compiler will emit position independent code.
+  ///
+  /// When set to `null`, the default behavior of the compiler will be used.
+  ///
+  /// Defaults to `true` for libraries and `false` for executables.
+  final bool? pic;
+
   CBuilder.library({
     required this.name,
     required this.assetId,
     this.sources = const [],
     this.dartBuildFiles = const ['build.dart'],
     @visibleForTesting this.installName,
+    this.pic = true,
   }) : _type = _CBuilderType.library;
 
   CBuilder.executable({
     required this.name,
     this.sources = const [],
     this.dartBuildFiles = const ['build.dart'],
+    this.pic = false,
```

Comment:
Maybe make this a named argument `pie` and assign to `this.pic` in the initializer list.

Reply:
Done